packages/binding-mqtt/src/mqtt-client.ts

```diff
@@ -40,7 +40,7 @@ export default class MqttClient implements ProtocolClient {
         this.scheme = "mqtt" + (secure ? "s" : "");
     }
 
-    private client: mqtt.MqttClient = undefined;
+    private client: mqtt.MqttClient | undefined = undefined;
```

Comment:
I think this could be simplified a bit :)

```suggestion
    private client?: mqtt.MqttClient;
```